README.md

```diff
@@ -135,6 +135,7 @@ To run as a desktop app:
 
    ```
    npm install electron
+   npm run install:electron
    node_modules/.bin/electron .
```

Comment:
I'm not sure why electron isn't a devDependency? That'd get rid of line #1
the last two could be replaced with `npm run electron` which has already been defined